### PR TITLE
espresso quick guide

### PR DESCRIPTION
**Pull request by aknackiron**
Initial Espresso guide based on [blog post](http://testdroid.com/news/how-to-use-espresso-v2-0-with-testdroid-cloud-devices)

### CODE REVIEW

File: espresso/index.md [hunk 1]
```diff
@@ -0,0 +1,89 @@
+---
+layout: default
+title: Espresso Testing
+---
+
+
+Testdroid Cloud has had Espresso support since it's first versions.
```
Comment (LasseHall) [inline espresso/index.md]: *its

File: espresso/index.md [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+
+
+   In addition, if your application uses functionalities from `espresso-contrib`, add the following line inside dependencies:
+
+     androidTestCompile 'com.android.support.test.espresso:espresso-contrib:2.0'
+
+5) Configure the instrumentation runner with the `testInstrumentationRunner`. It should look like this:
+
+     defaultConfig {
+       applicationId "com.example.vvhelppi.myapplication"
```
Comment (LasseHall) [inline espresso/index.md]: Should the example id be changed?

File: espresso/index.md [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+       minSdkVersion 15
+       targetSdkVersion 20
+       versionCode 1
+       versionName "1.0"
+       testInstrumentationRunner "android.support.test.runner.AndroidJUnit.Runner"
+     }
+
+Set Instrumentationrunner properlyMore information about [Espresso Core Resource can be found in this Javadoc](https://android-test-kit.googlecode.com/git/docs/javadocs/espresso/espresso-core-2.0-javadoc/reference/packages.html).
```
Comment (LasseHall) [inline espresso/index.md]: Is the beginning of sentence formatted as intended?

File: espresso/index.md [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+
+# The New Test Runner - AndroidJUnitRunner
+
+The **AndroidJUnitRunner** is a new unbundled test runner for Android, which is part of the Android Support Test Library and can be downloaded via the Android Support Repository. The new runner contains all improvements of GoogleInstrumentationTestRunner and adds more features, such as:
+
+* *JUnit4* support
+* Instrumentation Registry for accessing *Instrumentation*, *Context* and *Bundle Arguments*
+* Test Filters *@SdkSupress* and *@RequiresDevice*
```
Comment (LasseHall) [inline espresso/index.md]: is @SdkSupress correct or should it be @SdkSuppress?

File: espresso/index.md [hunk 5]
```diff
@@ -0,0 +1,89 @@
...
+
+# How to use Espresso v2.0 tests at Testdroid Cloud
+
+Let's take a look how to run your Espresso scripts at Testdroid Cloud. Follow these few steps and you should get your app up and running
+
+So, how you can run tests written with this cool new framework in Testdroid Cloud? Just few steps and you'll get your app tested with this framework on hundreds of real Android devices:
+
+## Step #1:
+
+First, you have to create a test configuration in Android Studio. Open **Run** -> **Edit Configurations**. Click '+' button on the left-top corner to add new configuration as **Android Tests**. Give a name to test configuration (here 'New Test Config'), select module (here 'app') and specify the instrumentation runner (here 'android.support.test.runner.AndroidJUnitRunner'). Your open Run/Debug Configuration dialog should look this this:
```
Comment (LasseHall) [inline espresso/index.md]: "dialog should look this this:"

File: espresso/index.md (hunk 1 above)
Comment (aknackiron) [inline espresso/index.md]: Darn copy paste...

**Pull request merged**